AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_materialproviders/smithygenerated/aws_cryptography_keystore/plugin.py

```diff
@@ -0,0 +1,44 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+from .config import Config, Plugin, smithy_config_to_dafny_config, KeyStoreConfig
+from smithy_python.interfaces.retries import RetryStrategy
+from smithy_python.exceptions import SmithyRetryException
+from .dafnyImplInterface import DafnyImplInterface
+
+
+def set_config_impl(config: Config):
+    """
+    Set the Dafny-compiled implementation in the Smithy-Python client Config
+    and load our custom NoRetriesStrategy.
+    """
+    config.dafnyImplInterface = DafnyImplInterface()
+    if isinstance(config, KeyStoreConfig):
+        from software_amazon_cryptography_keystore_internaldafny import default__
```

Comment:
It is wild since it is an anti-pattern: https://peps.python.org/pep-0008/#imports
This is done to avoid circular dependencies when importing generated Dafny code

Reply:
ah yeah i might have had a similar comment on a previous PR...